cmd/basin/publication.go

```diff
@@ -47,7 +49,7 @@ func newPublicationCommand() *cli.Command {
 }
 
 func newPublicationCreateCommand() *cli.Command {
-	var owner, dburi, provider string
+	var owner, dburi, provider, winSize string
```

Comment:
I think you could switch the `winSize` type from `string` to `int64`, in [L82](https://github.com/tablelandnetwork/basin-cli/pull/12/files#diff-73777451ff3bdcfcd6ee6970408801e8731a6368a183dfca6d1e1b7df2195fc7R82) use the `&cli.Int64Flag` struct, avoin the `parseInt` in [L127](https://github.com/tablelandnetwork/basin-cli/pull/12/files#diff-73777451ff3bdcfcd6ee6970408801e8731a6368a183dfca6d1e1b7df2195fc7R127)